app/assets/stylesheets/availabilities.scss

```diff
@@ -0,0 +1,82 @@
+// Place all the styles related to the availabilities controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+@import "colors";
+
+.dayname {
+  font-weight: bold;
+  vertical-align: middle;
+  text-align: center;
+  width: 4em;
+}
+
+table td {
+  touch-action: manipulation;
+  line-height: 40px;
+}
+
+.time-label {
+  display: block;
+  position: relative;
+  left: -25px;
+  font-weight: normal;
+}
+
+.availability-yes {
+  background-color: white;
+  border: 1px solid black;
+}
+
+.availability-no {
+  background-color: $light-grey;
+  border: 1px solid black;
+}
+
+.availability-yes-pending {
+  background-color: white;
+  border: 1px solid black;
+}
+
+.availability-no-pending {
+  background-color: $light-grey;
+  border: 1px solid black;
+}
+
+.white-text, .white-text:link, .white-text:visited, .white-text:hover, .white-text:active {
+  color: white;
+}
+
+.instruction {
+  display: inline-block;
+  float: right;
+  margin-right: 100px;
+}
+
+.box-no {
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  border: 1px solid black;
+  background-color: $light-grey;
+}
+
+.box-yes {
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  border: 1px solid black;
+  background-color: white;
+}
+
+.instruction-label {
+  display: inline-block;
+  margin-right: 20px;
+  margin-left: 5px;
+  position: relative;
+  top: -4px;
+}
+
+html {
```

Comment:
You can also somehow wrap this in a container maybe?

Reply:
tried to put the min-width to the div, but not working

Reply:
But anyways you should not put min-width in the html itself. Let me try and fix